### PR TITLE
:sparkles: add support for generated documents 

### PR DESCRIPTION
**Pull request by ianardee**

### CODE REVIEW

File: src/Mindee/Mindee.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net7.0;net8.0;net472</TargetFrameworks>
+    <TargetFrameworks>net472;net48;net6.0;net7.0;net8.0</TargetFrameworks>
```
Comment (sebastianMindee) [inline src/Mindee/Mindee.csproj:4]: This syntax is so cursed btw

File: src/Mindee/MindeeClientOptions.cs [hunk 2]
```diff
@@ -54,16 +54,26 @@ public sealed class AsyncPollingOptions
...
+        public int IntervalMilliSec { get; }
+
         /// <summary>
         /// Options to pass for asynchronous parsing with polling.
         /// </summary>
         /// <param name="initialDelaySec"><see cref="InitialDelaySec"/></param>
         /// <param name="intervalSec"><see cref="IntervalSec"/></param>
         /// <param name="maxRetries"><see cref="MaxRetries"/></param>
-        public AsyncPollingOptions(double initialDelaySec = 4.0, double intervalSec = 2.0, int maxRetries = 30)
+        public AsyncPollingOptions(double initialDelaySec = 4.0, double intervalSec = 1.5, int maxRetries = 60)
```
Comment (sebastianMindee) [inline src/Mindee/MindeeClientOptions.cs:73]: Other defaults are 2s & 60 retries (2 minutes total). I don't mind changing delays on other libs, but I don't really like the timers being different on each

File: tests/Mindee.IntegrationTests/MindeeClientTest.cs [hunk 3]
```diff
@@ -44,12 +50,10 @@ public async Task Parse_File_SinglePage_MustSucceed()
         }
 
         [Fact]
-        public async Task Parse_Url_SinglePage_MustSucceed()
+        public async Task Parse_Url_Standard_SinglePage_MustSucceed()
         {
-            var apiKey = Environment.GetEnvironmentVariable("Mindee__ApiKey");
-            var mindeeClient = new MindeeClient(apiKey);
             var inputSource = new UrlInputSource("https://raw.githubusercontent.com/mindee/client-lib-test-data/main/products/expense_receipts/default_sample.jpg");
```
Comment (sebastianMindee) [inline tests/Mindee.IntegrationTests/MindeeClientTest.cs:55]: Never noticed this one. I think I'll steal it :smiling_imp:

Comment (ianardee) [reply tests/Mindee.IntegrationTests/MindeeClientTest.cs:55]: :+1:

File: src/Mindee/MindeeClientOptions.cs (hunk 2 above)
Comment (ianardee) [reply src/Mindee/MindeeClientOptions.cs]: Was doing some testing. Good catch, reverted.

File: src/Mindee/Geometry/Polygon.cs [hunk 4]
```diff
@@ -96,15 +96,7 @@ public bool IsPointInY(Point point)
...
-            StringBuilder result = new StringBuilder("[\n");
-            for (int i = 0; i < this.Count; i++)
-            {
-                result.Append($"  {this[i]}");
-                if (i >= this.Count - 1) result.Append('\n');
-                else result.Append(",\n");
-            }
-            result.Append(']');
-            return result.ToString();
+            return $"Polygon with {this.Count} points.";
```
[the remark this answers is not in the document]
Comment (ianardee) [reply src/Mindee/Geometry/Polygon.cs]: fixed

**Review by sebastianMindee — APPROVED** [review]

**Pull request merged**